src/main/java/uk/gov/hmcts/divorce/divorcecase/search/SearchInputFields.java

```diff
@@ -59,11 +59,25 @@ public class SearchInputFields implements CCDConfig<CaseData, State, UserRole> {
         SearchField.<UserRole>builder().label("Applicant first name").id(APPLICANT_1_FIRST_NAME).build(),
         SearchField.<UserRole>builder().label("Applicant last name").id(APPLICANT_1_LAST_NAME).build(),
         SearchField.<UserRole>builder().label("Applicant email").id(APPLICANT_1_EMAIL).build(),
-        SearchField.<UserRole>builder().label("Applicant postcode").id(APPLICANT_1_ADDRESS).listElementCode("PostCode").build(),
+        SearchField.<UserRole>builder().label("Applicant postcode")
```

Comment:
I don't understand this bit - with the caseworker and judge roles only

Reply:
superuser and LA are usually granted on top of the caseworker role, and I believe this should allow all professional users to search by address except for sols